week-1/mandatory/2-classes-db/task.md

```diff
@@ -2,13 +2,21 @@
 
 ## Submission
 
-Below you will find a set of tasks for you to complete to consolidate and extend your learning from this week.  You will find it beneficial to complete the reading tasks before attempting some of these.
+Below you will find a set of tasks for you to complete to consolidate and extend your learning from this week. You will find it beneficial to complete the reading tasks before attempting some of these.
 
 To submit this homework write the correct commands for each question here:
 
 ```sql
-
-
+1. select * from rooms where rate > 100;
+2. select * from reservations where (checkin_date >= '2020-09-01' and checkin_date <='2020-09-30') and checkout_date - checkin_date < 3;
+3. select * from customers where city like 'M%';
+4. insert into room_types values('PENTHOUSE','185.00');
+5. insert into rooms (room_no, rate, room_type) values(501,(select def_rate from room_types where room_type = 'PENTHOUSE'),'PENTHOUSE'),(502,(select def_rate from room_types where room_type = 'PENTHOUSE'),'PENTHOUSE');
+6. insert into rooms (room_no, rate, room_type) values(503,143.00,'PREMIER PLUS');
+7. select all from reservations where checkin_date >= '2020-08-01' and checkin_date <= '2020-08-31'; / select count(id) from reservations where checkin_date >= '2020-08-01' and checkin_date <= '2020-08-31';
+8. select sum(checkout_date - checkin_date) from reservations where room_no between 200 and 299;
+9. select sum(total),avg(total) from invoices where total > 300;
+10. select sum(case when room_no between 100 and 199 then checkout_date - checkin_date end) as first_floor, sum(case when room_no between 200 and 299 then checkout_date - checkin_date end) as second_floor, sum(case when room_no between 300 and 399 then checkout_date - checkin_date end) as third_floor from reservations;
```

Comment:
Wow!  Very innovative and interesting use of the CASE construct!  Very good - you've clearly done more reading than was asked for - well done.

Reply:
Thanks a lot